independent-projects/bootstrap/pom.xml

```diff
@@ -24,6 +24,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <!-- Versions -->
+        <javax.json.version>1.1.4</javax.json.version>
```

Comment:
Not required, but if you want a JSON Writer with less dependencies (specially on the javax.json API), there is https://github.com/ralfstx/minimal-json